pkgs/applications/misc/bitwarden-menu/default.nix

```diff
@@ -1,23 +1,30 @@
 { lib
 , buildPythonApplication
+, python3Packages
 , fetchPypi
 , pynput
-, xdg
+, xdg-base-dirs
 }:
 
 buildPythonApplication rec {
   pname = "bitwarden-menu";
-  version = "0.4.1";
-  format = "setuptools";
+  version = "0.4.3";
+  format = "pyproject";
```

Comment:
This should be pyproject = true

Reply:
I'll make this change.. but tomorrow..

Where can i look up the proper documentation for packaging python ..

Reply:
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters